### PR TITLE
 makuring.com #103031

### PR DESCRIPTION
**Pull request by SKEIDs**
#103031

### CODE REVIEW

File: AnnoyancesFilter/sections/self-promo.txt [hunk 1]
```diff
@@ -50,6 +50,9 @@ disk.yandex.*##.mail360-banner
 !-----------------------------------------------
 !
 !
+!
+! https://github.com/AdguardTeam/AdguardFilters/issues/103031
+makuring.com##.shtb2.tbrsp
```
Comment (zloyden) [inline AnnoyancesFilter/sections/self-promo.txt:55]: You'll block their images. I'd use
```
makuring.com#?#.sponsored:has(> div.textwidget > div[class] > div.cell > a[href^="https://www.youtube.com/channel/"])
```
for the `Annoyances - subscriptions`, but I need a review by @Yuki2718.
Also add `makuring.com##.center > p.nospace` to the `AdGuard Japanese - specific`.

**Review by zloyden — CHANGES_REQUESTED** [review]

File: AnnoyancesFilter/sections/self-promo.txt (hunk 1 above)
Comment (SKEIDs) [reply AnnoyancesFilter/sections/self-promo.txt]: ok.

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/self-promo.txt (hunk 1 above)
Comment (Yuki2718) [reply AnnoyancesFilter/sections/self-promo.txt]: These are links to thier social media and are not redundant. We don't block them in principle, do we @zloyden ?

Comment (Yuki2718) [reply AnnoyancesFilter/sections/self-promo.txt]: Well, found redundant links at the bottom. Hmm...

Comment (zloyden) [reply AnnoyancesFilter/sections/self-promo.txt]: They have no recommendations, just social links? If so, we should remove the rule.

Comment (zloyden) [reply AnnoyancesFilter/sections/self-promo.txt]: We remove social links if they have:
1. Recommendations
2. Aggressive placement in the content and they also duplicate links on the header / footer.

**Review by zloyden — APPROVED** [review]

File: AnnoyancesFilter/sections/self-promo.txt (hunk 1 above)
Comment (Yuki2718) [reply AnnoyancesFilter/sections/self-promo.txt]: @SKEIDs Please remove `akuring.com#?#.sponsored:has(> div.textwidget > div[class] > div.cell > a[href^="https://www.youtube.com/channel/"])`.

**Review by Yuki2718 — APPROVED** [review]

**Comment by SKEIDs** [comment]
@zloyden @Yuki2718 
I am sorry. and thank you for help.

**Pull request merged**